From 8d2d7d8b9d5f5d5ec3ed8241ff93654f1ebb8973 Mon Sep 17 00:00:00 2001
From: Boian Bonev <bbonev@ipacct.com>
Date: Tue, 28 Sep 2021 03:26:08 +0300
Subject: [PATCH] better install target
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

- remove -T from install for busybox compat (Milan P. StaniÄ‡ <mps@arvanta.net>)
- also install the man page
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 54d6cbb..a629710 100644
--- a/Makefile
+++ b/Makefile
@@ -81,7 +81,8 @@ clean:
 	rm -f bpfmon bpfmon.o psort psort.o
 
 install: bpfmon
-	$(INSTALL) -TD -m 0755 $< $(DESTDIR)$(PREFIX)/sbin/$<
+	$(INSTALL) -D -m 0755 $< $(DESTDIR)$(PREFIX)/sbin/$<
+	$(INSTALL) -D -m 0644 bpfmon.8 $(DESTDIR)$(PREFIX)/share/man/man8/bpfmon.8
 	$(STRIP) $(DESTDIR)$(PREFIX)/sbin/$<
 
 mkotar:
